hosts/ixp-as11201/data.nix

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Suggestions:
- `cortex.nix`
- `net.nix`
- `vm.nix`

Reply:
It's not vm (see hv01) or networking (see #87) specific.